js/components/portal-dashboard/question-area.tsx

```diff
@@ -40,7 +40,7 @@ export class QuestionArea extends React.PureComponent<IProps>{
               <LaunchIcon className={css.icon} />
             </a>
             {hasTeacherEdition &&
-              <a className={css.teacherEditionIcon} href={activityURL+"?mode=teacher-edition"} target="_blank" data-cy="open-teacher-edition-button">
+              <a className={css.teacherEditionIcon} href={activityURL+"/?mode=teacher-edition"} target="_blank" data-cy="open-teacher-edition-button">
```

Comment:
I would still write this with spacing so we have consistent style.  Either
```
<a className={css.teacherEditionIcon} href={activityURL + "/?mode=teacher-edition"} target="_blank" data-cy="open-teacher-edition-button">
```
or
```
<a className={css.teacherEditionIcon} href={`${activityURL}/?mode=teacher-edition`} target="_blank" data-cy="open-teacher-edition-button">
```